corehq/apps/app_manager/models/common.py

```diff
@@ -2186,6 +2143,8 @@ def uses_usercase(self):
 
 
 class AdvancedForm(IndexedFormBase, NavMenuItemMediaMixin):
+    from corehq.apps.app_manager.models.schedules import FormSchedule
```

Comment:
It's a shame that these all became circular imports, but I don't see an easy way around it with this pattern. I wonder if you were to inline the imports in `schedule.py` rather than here, it might clean up `common.py` a bit more, which I think will probably get read more often.